FileSystem/FileIOUtil.js

```diff
@@ -212,3 +212,162 @@ function getLineStart(file) {
   }
   return filePointer + 1;
 }
+
+/**
+ * Helper function to parse one variable
+ * @param(object) file File structure
+ * @param {VbaBox} variable reference variable to store value
+ */
+function inputFileUtil(file, variable) {
+  var content = file.content;
+  var filePointer = file.pointer;
+  // Jump over whitespace
+  while (filePointer < content.length && content[filePointer] == ' ') {
+    filePointer++;
+  }
+  // No input left
+  if (filePointer == content.length) {
+    throw new Error('End of file reached');
+  }
+  file.pointer = filePointer;
+  // Try obtaining an input
+  if (!(tryInputConstant(file, variable) || tryInputVariable(file, variable))) {
+    throw new Error('Unknown Expression');
+  }
+  filePointer = file.pointer;
+  // Jump over whitespace and break at any delimiting characters
+  while (filePointer < content.length) {
```

Comment:
Can we club similar conditions, will be more readable.

Reply:
Done, reduced 5 cases to 3.